index.html

```diff
@@ -243,26 +243,43 @@ <h3>Sampling and Reporting Rate</h3>
 
 <section> <h2>Platform primitives</h2>
   <p>
-    The [=platform collector=] refers to a platform interface, with which the [=user agent=] interacts to
-    obtain the telemetry readings required by this specification.
+    A <dfn>pressure source</dfn> is an abstract, [=implementation-defined=]
+    interface to hardware counters or an underlying telemetry framework that
+    provides information about a <dfn>source types</dfn>
+    defined by {{PressureSource}}. A [=pressure source=] can make use of data
+    fusion with data from additional sources if that provides better results.
   </p>
   <p>
-    A [=platform collector=] can be defined by the underlying platform (e.g. in a native telemetry
-    framework) or by the [=user agent=], if it has a direct access to hardware counters.
+    A <dfn>platform collector</dfn> is an abstract interface responsible for
+    obtaining telemetry samples from a [=pressure source=], translating them
+    into [=pressure states=] and providing them to the [=user agent=].
   </p>
   <p>
-    A [=platform collector=] can support telemetry for different <dfn>source types</dfn> of computing
-    devices defined by {{PressureSource}}, or there can be multiple [=platform collectors=].
+    A [=platform collector=] has the following associated data:
+  </p>
+  <ul>
+    <li>
+      an <dfn data-dfn-for="platform collector">associated pressure
+      source</dfn>, which is a [=pressure source=] or null.
+    </li>
+    <li>
+      an <dfn data-dfn-for="platform collector">activated</dfn> boolean flag,
+      initially false.
+    </li>
+  </ul>
+  <p>
+    The format of the data provided by a [=pressure source=] is
+    [=implementation-defined=], and so is the process through which a
+    [=platform collector=] translates it into a [=pressure state=].
   </p>
   <p>
-    From the implementation perspective [=platform collector=] can be treated as a software proxy for the
-    corresponding hardware counters. It is possible to have multiple [=platform collector=] simultaneously
-    interacting with the same underlying hardware if the underlying platform supports it.
+    For this specification's purposes, [=platform collectors=] are scoped to a
+    [=global object=] via the [=platform collector mapping=].
   </p>
   <p>
-    In simple cases, a [=platform collector=] represents individual hardware counters, but if the provided
-    counter readings are a product of data fusion performed in software, the [=platform collector=]
-    represents the results of the data fusion process. This may happen in user space or in kernel space.
+    For automation purposes, a [=platform collector=] must have the ability to connect to
+    [=virtual pressure sources=] and use their simulated data as already [=adjusted pressure states=]
```

Comment:
already states sounds weird.

Reply:
Reworded in b11010f1d1dab6077737011110bce5e3c5acac19